module/zfs/dmu_send.c

```diff
@@ -1945,6 +1946,14 @@ setup_featureflags(struct dmu_send_params *dspp, objset_t *os,
 		*featureflags |= DMU_BACKUP_FEATURE_LZ4;
 	}
 
+	/* XXX: Allan: should be able to use embedok without implying ZSTD */
```

Comment:
This needs cleaning

Reply:
I left it there for now as I don't know if there's anything to fix

Reply:
i removed it from my tree since @Ornias1993 wanted it that way. but personally i think. better leave a useless comment, than removing a important one

Reply:
unfortunatly  the comment doesnt explain much about the background. so nobody understands what he is talking about. i somewhat understand it in a foggy way.

Reply:
Comments should always be clear, they are not for personal use but to instruct future coders.

Reply:
> unfortunatly the comment doesnt explain much about the background. so nobody understands what he is talking about. i somewhat understand it in a foggy way.

> Comments should always be clear, they are not for personal use but to instruct future coders.

> I didn't want it "away", I want it either clearified or away... thats a significant difference.


Well, **can** we clarify it? if not, I'll drop it

Reply:
I honestly have no idea what he is talking about.